examples/Powershell Script/readme.md

```diff
@@ -7,21 +7,26 @@ For powershell scripts to run successfully, the execution policy needs to be set
 <strong>Command Line Monitor - Powershell Command Embedded File</strong>
 
 Command run on host:
+```bash
 @powershell.exe -inputformat none %{FILE:samplepowershell.ps1} test example result
```

Comment:
As this is Windows, try `bat` or `batchfile` instead of `bash`

You could verify the rendering of the different options by editing the Markdown files directly in GitHub (I won't mind).

Also, more details about Markdown on GH: https://docs.github.com/en/github/writing-on-github/working-with-advanced-formatting/creating-and-highlighting-code-blocks